### ISSUE TITLE
Friends activity no longer works

### ISSUE THREAD

**Issue by jorp** [ValveSoftware/steam-for-linux#11297]
#### Your system information

* Steam client version: 1726683985
* SteamOS version: 3.5.19
* Opted into Steam client beta?:  No
* Opted into SteamOS beta?: No 
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:

When first booting up a Steam Deck on the above software version, scrolling down from  'Friends' to 'Activity'  no longer shows any information.

See relevant threads here:
https://steamcommunity.com/app/1675200/discussions/1/4759830079702867948
https://steamcommunity.com/app/1675200/discussions/1/6895657216190544579/
https://steamcommunity.com/app/1675200/discussions/1/4759830079701214851/
https://steamcommunity.com/app/1675200/discussions/1/4425436358665905430/
https://steamcommunity.com/app/1675200/discussions/1/4513254345531880083/

**Comment by TTimo** [comment]
Can you reproduce this issue after opting into the Steam client beta and SteamOS beta?

**Comment by jorp** [comment]
> Can you reproduce this issue after opting into the Steam client beta and SteamOS beta?

No, it's still not working on beta. Anything else I could try or additional information I could provide? 

Steam Beta Branch: Steam Deck Beta
Steam Version: 1730253536

**Comment by ruifcastro** [comment]
I have been able to fix it but it still happens time to time.

Here's my steps:
- go to desktop mode
- open Steam app and exit it (File->Exit)
- go to folder /home/deck/.local/share/Steam/ and cut (move) the folder "userdata" to another location. I moved to my desktop
- now start Steam again
- a new "userdata" folder should have been created
- I exited the Steam app again
- i moved the "userdata" on my desktop back to /home/deck/.local/share/Steam/ and clicked the apply all and clicked the Skip so it didn't overwrite the newly created clean files.
- go back to Game Mode (Gamescape)

Your Activity Tab should be working again. In reality it's only the config folder: /home/deck/.local/share/Steam/userdata/number of your account/config
I'm still not sure what triggers this. The closest I've got is the none steam shortcuts seem to me to be my problem. If I don't add them the Activity Tab seems to work well. But I'm stiil not sure because if I copy the same file shortcuts.vdf again back to the config folder. It doesn't seem to stop working.

Any ways, with the fix above there are Side Effects:
- battery setting to show percentage has to be set again.
- non-steam games lose shortcuts
- steam initial tutorials are active again
- some games can ask login and password again
- i think you also lose the performance profiles for each game. Not sure.

**Comment by jorp** [comment]
Just wanted to add that this is still an issue on 3.7.17
